### PR TITLE
Add intel Alder Lake BIOS leak to compromises

### PR DESCRIPTION
**Pull request by Caze121**

### CODE REVIEW

**Comment by Caze121** [comment]
Fixed the spelling, and all changes are verified.

**Review by sublimino — APPROVED** [review]
LGTM, thanks for the contribution 🙏

**Comment by sublimino** [comment]
@Caze121 ah, commit message formatting needs updating please

**Comment by szh** [comment]
@Caze121 Take a look at [this page](https://github.com/cncf/tag-security/pull/993/checks?check_run_id=9121532172) for a list of commands to run that will fix the formatting.

**Comment by Caze121** [comment]
I rebased with --signoff and then pushed via GitHub Desktop.  Now I'm getting a DCO failure on the Merge branch step.

**Comment by szh** [comment]
@Caze121 I would recommend you first [squash](https://stackoverflow.com/a/5189600) all your commits into one. Then just ensure that the commit message contains the string
`Signed-off-by: Ryan Gibbons <ryanjgibbons@hotmail.com>`

**Comment by Caze121** [comment]
Yes, finally!  Thank you for the help

File: supply-chain-security/compromises/2022/intel-alder-lake-BIOS-leak.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+"...Security researcher Mark Ermolov has already been hard at work analyzing the code. His early reports indicate that he has found secret MSRs (Model Specific Registers) that are typically reserved for privileged code and thus can present a security problem, along with the private signing key used for Intel's Boot Guard, thus potentially invalidating the feature. In addition, there are also signs of ACMs (Authenticated Code Modules) for BootGuard and TXT (Trusted Execution Technology), portending potential future issues with the root of trust."
+
+## Type of Compromise
+
+This incident fits the [Source Code](../compromise-definitions.md#dev-tooling)
+definition.
+
+## References
+
+- [Intel Confirms Alder Lake BIOS Source Code Leak](https://www.tomshardware.com/news/intel-confirms-6gb-alder-lake-bios-source-code-leak-new-details-emerge)#
```
Comment (szh) [inline supply-chain-security/compromises/2022/intel-alder-lake-BIOS-leak.md:25]:
```suggestion
- [Intel Confirms Alder Lake BIOS Source Code Leak](https://www.tomshardware.com/news/intel-confirms-6gb-alder-lake-bios-source-code-leak-new-details-emerge)
```

I assume the trailing `#` is a typo?

File: supply-chain-security/compromises/2022/intel-alder-lake-BIOS-leak.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Impact
+
+"Now that the BIOS/UEFI code is in the wild and Intel has confirmed it as legitimate, both nefarious actors and security researchers alike will undoubtedly probe it to search for potential backdoors and security vulnerabilities."
+
+"...Security researcher Mark Ermolov has already been hard at work analyzing the code. His early reports indicate that he has found secret MSRs (Model Specific Registers) that are typically reserved for privileged code and thus can present a security problem, along with the private signing key used for Intel's Boot Guard, thus potentially invalidating the feature. In addition, there are also signs of ACMs (Authenticated Code Modules) for BootGuard and TXT (Trusted Execution Technology), portending potential future issues with the root of trust."
+
+## Type of Compromise
+
+This incident fits the [Source Code](../compromise-definitions.md#dev-tooling)
```
Comment (szh) [inline supply-chain-security/compromises/2022/intel-alder-lake-BIOS-leak.md:20]:
```suggestion
This incident fits the [Source Code](../compromise-definitions.md#source-code)
```

**Comment by Caze121** [comment]
Can't figure out how to get a signed merge through the GitHub Desktop client.  Will abandon and see if I can use another way.

**Comment by Caze121** [comment]
Thanks for the help!  I'm trying to figure out why my merges fail the DCO check each time.  I am using GitHub Desktop to submit but can't see how to sign them.  Should I just switch to another method?

**Comment by Caze121** [comment]
Thanks for the help!  I'm trying to figure out why my merges fail the DCO check each time.  I am using GitHub Desktop to submit but can't see how to sign them.  Should I just switch to another method?

**Comment by szh** [comment]
@Caze121 I started a new branch off of main and copied your changes over. See #1006 
Please review and let me know if I got it right.

**Pull request closed without merging**